samples/modules/thrift/hello/README.rst

```diff
@@ -57,12 +57,23 @@ Requirements
    west update
 
 - QEMU Networking (described in :ref:`networking_with_qemu`)
-- Thrift dependencies installed for your host OS e.g. in Ubuntu
+- Thrift dependencies installed for your host OS
 
-.. code-block:: console
-   :caption: Install additional dependencies in Ubuntu
+.. tabs::
+
+   .. group-tab:: Ubuntu
+
+      .. code-block:: bash
+        :caption: Install thrift dependencies in Ubuntu
+
+         sudo apt install -y libboost-all-dev thrift-compiler libthrift-dev
+
+   .. group-tab:: macOS
+
+      .. code-block:: bash
+        :caption: Install thrift dependencies in macOS
```

Comment:
```suggestion
        :caption: Install thrift dependencies on macOS
```

Reply:
Hmm.. do people normally say a package is installed "on" an OS? I usually say "in".

It's possibly "on" in German or French, but I've been saying "in" for the last 20 years or so and I'm a native English speaker.

Reply:
Oh. Could well be that you're right then. I say "on" but I'm not a native English speaker.